### ISSUE TITLE
Problems with running custom datasets

### ISSUE THREAD

**Issue by xzr52** [yiqun-wang/PET-NeuS#3]
This is a very great job, but when I run the custom dataset, the cameras.npz file is generated according to the neus format, but the effect of reconstructing several scenes is not good. Do you know why
![image](https://github.com/yiqun-wang/PET-NeuS/assets/56352290/228b5827-739a-4ef6-b077-202b5aa313a1)
![image](https://github.com/yiqun-wang/PET-NeuS/assets/56352290/fe6c1c11-ed58-4ca5-b625-e0f3aee2f74b)
![Uploading image.png…]()
And the picture I used for training looks like this
![IMG20220607103013](https://github.com/yiqun-wang/PET-NeuS/assets/56352290/14469c56-d892-462a-9dd4-50027e947415)
![IMG_4565](https://github.com/yiqun-wang/PET-NeuS/assets/56352290/85da80a6-abb6-4f3d-be5d-e02d1111e5b0)
If you have a good solution, I will be very grateful for your help

**Comment by yiqun-wang** [comment]
Hello, have you tried to modify the configuration file by changing the value of use_white_bkgd ?

**Comment by xzr52** [comment]
> 您好，您是否尝试过通过更改use_white_bkgd的值来修改配置文件？

Yes,when i choose to use_white_bkgd,i also get bad results.And i just change the batch_size.
![image](https://github.com/yiqun-wang/PET-NeuS/assets/56352290/46d16253-63da-43c2-b8e0-f1d11cd22f22)
![image](https://github.com/yiqun-wang/PET-NeuS/assets/56352290/7f5bccac-d3cc-44d3-9672-3968a389690d)
![image](https://github.com/yiqun-wang/PET-NeuS/assets/56352290/80675534-ca01-42bb-b7ee-87e9c45096bc)
if you have any resulotion,i would be very grateful. Thank you for your answer.

**Comment by xzr52** [comment]
some scenes have rendered all white when i set use_white_bkgd=True
![image](https://github.com/yiqun-wang/PET-NeuS/assets/56352290/1b6e8f0b-bcde-4c36-b3d1-8d3ee2f02ce5)
if i  set use_white_bkgd=false it like that:
![image](https://github.com/yiqun-wang/PET-NeuS/assets/56352290/35698a0d-6cc6-4679-b585-ac67b10984c7)
but have bad mesh in one sphere:
![image](https://github.com/yiqun-wang/PET-NeuS/assets/56352290/50add1b5-ddbb-49d2-9394-a92816f72955)

**Comment by yiqun-wang** [comment]
Where is the mesh from? In the function of validate_mesh, the default resolution is just 64.

**Comment by xzr52** [comment]
> 网格来自哪里？在validate_mesh函数中，默认分辨率仅为64。

This is the mesh it evaluates when I train
![image](https://github.com/yiqun-wang/PET-NeuS/assets/56352290/e96edd2f-c0fa-4107-bc7c-87614b4490a0)
